tutorials/site-monitoring-foundations.ipynb

```diff
@@ -0,0 +1,2737 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        chunks={}
</pre><p>I would add <code>dtype="int16"</code> here to reduce the memory footprint of the array. I am pretty sure the Sentinel 2 data are integer data types. The STAC metadata does not declare it :( so odc-stac just assumes float.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/PlanetaryComputerExamples/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>